framework/src/base/Assembly.C

```diff
@@ -2165,8 +2154,6 @@ Assembly::computeADFace(const Elem * elem, unsigned int side)
 
   if (_subproblem.haveADObjects())
   {
-    const std::unique_ptr<const Elem> side_elem(elem->build_side_ptr(side));
```

Comment:
@lindsayad you say wat?!

Build an element side for _every_ AD face calc even without displaced mesh

Reply:
👍